natchez-extras-testkit/src/main/scala/com/ovoenergy/natchez/extras/testkit/TestEntryPoint.scala

```diff
@@ -46,6 +47,11 @@ object TestEntryPoint {
               def spanId: F[Option[String]] = F.pure(None)
               def traceUri: F[Option[URI]] = F.pure(None)
               def kernel: F[Kernel] = F.pure(kern)
+              override def log(fields: (String, TraceValue)*): F[Unit] = put(fields: _*)
```

Comment:
It's OK to add the `override` keyword as this is something we might want to do in the future, but we need to be consistent, so either add it everywhere in this file, or we don't add it at all.